tests/foreman/cli/test_logging.py

```diff
@@ -0,0 +1,278 @@
+"""CLI tests for logging.
+
+:Requirement: Logging
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: logging
+
+:TestType: Functional
+
+:CaseImportance: Medium
+
+:Upstream: No
+"""
+import re
+import logging
+from fauxfactory import gen_string
+from robottelo import ssh
+from robottelo.ssh import upload_file
+from robottelo import manifests
+from robottelo.test import CLITestCase
+from robottelo.cli.factory import (
+    make_org,
+    make_product_wait,
+    make_repository,
+)
+from robottelo.cli.subscription import Subscription
+log = logging.getLogger(__name__)
+
+
+class SimpleLoggingTestCase(CLITestCase):
+    """Test class for default logging to files."""
+
+    org = None
+    product = None
+
+    @classmethod
+    def setUpClass(cls):
+        """Tests for logging to files"""
+        super(SimpleLoggingTestCase, cls).setUpClass()
+
+        # need own org for the manifest refresh test
+
+        cls.org = make_org(cached=True)
+        cls.product = make_product_wait(
+            {u'organization-id': cls.org['id']},
+            )
+
+    def _make_repository(self, options=None):
+        """Makes a new repository and asserts its success"""
+        if options is None:
+            options = {}
+
+        if not options.get('product-id'):
+            options[u'product-id'] = self.product['id']
+
+        return make_repository(options)
+
+    def test_positive_logging_from_foreman_core(self):
+        """Check that GET command to Hosts API is logged and has request ID.
+
+        :id: 0785260d-cb81-4351-a7cb-d7841335e2de
+
+        :expectedresults: line of log with GET has request ID
+
+        :CaseImportance: Medium
+        """
+
+        GET_line_found = 0
```

Comment:
Can you make these boolean?

```python
GET_line_found = False
....
# in for loop:
GET_line_found = True
....
# after for loop:
assert GET_line_found, "failure message"  # or "assert GET_line_found == True", if you want to be very explicit
```

Reply:
flake8 does not like this form "assert GET_line_found == True"
tests/foreman/cli/test_logging.py:104:31: E712 comparison to True should be 'if cond is True:' or 'if cond:'

But this format works: "assert GET_line_found,"

All changed.
Thank you